src/core/drive/form_submission.ts

```diff
@@ -64,7 +64,7 @@ export class FormSubmission {
   }
 
   get action(): string {
-    return this.submitter?.getAttribute("formaction") || this.formElement.action
+    return this.submitter?.getAttribute("formaction") || this.formElement.getAttribute("action")
```

Comment:
```suggestion
    return this.submitter?.getAttribute("formaction") || this.formElement.getAttribute("action") || this.formElement.action
```

<img width="535" alt="Screenshot 2021-06-04 at 18 54 29" src="https://user-images.githubusercontent.com/72359485/120843919-96aa2080-c566-11eb-8bcf-30dcb8fd4737.png">